corehq/apps/users/views/mobile/users.py

```diff
@@ -851,6 +852,8 @@ def _modify_user_status(request, domain, user_id, is_active):
         })
     user.is_active = is_active
     user.save(spawn_task=True)
+    log_model_change(request.user, user.get_django_user(), message={'is_active': is_active},
```

Comment:
```suggestion
    log_model_change(request.user, user.get_django_user(), fields_changed={'is_active': is_active},
```

Reply:
@snopoke i thought of that too :) , but `fields_changed` is a list of fields that has changed. It does not have the value the fields were assigned,

Reply:
does this even work? Looking at the code I don't see how message can be anything other than a string or a list

Reply:
Yes @snopoke , this does work. `log_model_change` itself passes a [dict message](https://github.com/dimagi/commcare-hq/blob/1256dc7af10e9704532ccaf037d2bb5f9741edb8/corehq/util/model_log.py#L36).

Reply:
I think that's a bug, it's saving to a charfield on the model. `LogEntry.objects.log_action` check if it's a list and serializes it to json if it is. How do these render on the UI as it is now?

Reply:
an example attached in screenshot in https://github.com/dimagi/commcare-hq/pull/28058

Reply:
this is how it should render: 
![image](https://user-images.githubusercontent.com/249606/87963868-b4d77100-cab9-11ea-8ef1-5523e724012c.png)

Reply:
```suggestion
    log_model_change(request.user, user.get_django_user(), message=f"Changed 'is_active' to {is_active}",
```